File: ThematicMapCreator.Api/Contracts/LayerStyleOptions/SimpleStyleOptions.cs
using System;

namespace ThematicMapCreator.Api.Contracts.LayerStyleOptions
{
    public class SimpleStyleOptions : ILayerStyleOptions
    {
        public string Color { get; set; }
        public string FillColor { get; set; }
        public double Size { get; set; }
        public LayerStyle Style { get; } = LayerStyle.None;
    }
}

File: Sources/ThematicMapCreator.Client/src/app/models/layer-style-options/simple-style-options.ts
import { LayerStyleOptions } from './layer-style-options';
import { LayerStyle } from './layer-style.enum';

export class SimpleStyleOptions implements LayerStyleOptions {
    style: LayerStyle.None;
    size = 8;
    color = '#000000';
    fillColor = '#000000';
}
